script/Arcueid/Day 5/05_00_ARC05_1B.txt

```diff
@@ -422,7 +425,7 @@ I bring my mouth close to her ear so that I can clearly get my point across.
 [sha:9bc3514aacab9dbeffb87ea14b5828b2c1c9f62e]{
 -- Page 70, Offset 8757.
 -- 「……あのな、アルクェイド」
-"...You know, Arcueid."
+"You know, Arcueid..."
```

Comment:
I think this one is a case where I prefer ... at the start because he's slowly leading into it